### DISCUSSION TITLE
How to compress API response and static asset with brotli?

### DISCUSSION THREAD

**Discussion by itsmnthn** [nitrojs/nitro#390] [Q&A] [answered]
I'm working on API using `nitro` and I have requirement to compress it with `brotli` compression to save bandwidth, the API response which is mostly static and static asset which will likely to change in years.

**Comment by Hebilicious** [comment, marked as answer]
AFAIK this is achieved differently for each provider. I assume that from the Nitro side, you just have to specify the correct header.
